modules/structs/repo_file.go

```diff
@@ -71,11 +71,11 @@ type ChangeFileOperation struct {
 	// enum: create,update,delete
 	Operation string `json:"operation" binding:"Required"`
 	// path to the existing or new file
-	Path string `json:"path" binding:"MaxSize(500)"`
-	// content must be base64 encoded
 	// required: true
+	Path string `json:"path" binding:"Required;MaxSize(500)"`
+	// new or updated file content, must be base64 encoded
 	Content string `json:"content"`
```

Comment:
Wait, `Content` is no longer required?

Reply:
You don't need content for file deletion